SU-SWS/stanford_media
# HSD8-129 Call video plugins to get video title for entity name

# READY FOR REVIEW

# Summary
- Populate embed code titles based on bundle suggestion plugins. for videos, it then calls the video provider plugins to get the titles. see https://www.drupal.org/project/video_embed_field/issues/2913925
- Fixed max file size issue when media bundle is configured to prevent certain file sizes.

# Needed By (Date)
- Some Day

# Urgency
- low

# Steps to Test

1. checkout latest (suhumsci repo)[https://github.com/SU-HSDO/suhumsci]
1. composer update/install `composer install --prefer-source`
1. sync a site `blt drupal:sync --site=[sitename]`
1. in any wysiwyg click the media browser button.
1. on the embed tab, enter a youtube url
1. after saving, ensure the entity title is the same as the youtube title.

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
